src/index.js

```diff
@@ -0,0 +1,41 @@
+var request = require('request');
+var unzip = require('unzip');
+var csv2 = require('csv2');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+
+
+var domains = [];
+
+mkdirp('output', function (err) {
```

Comment:
This is picky af, but `mkdirp` can be used synchronously like `mkdirp.sync(dir, opts)`. It may be better to make this check first. It would save putting that entire blob of code in a callback.

Reply:
I completely agree, I've made the requested alterations.